web/src/components/Upstream/UpstreamForm.tsx

```diff
@@ -771,7 +775,12 @@ const UpstreamForm: React.FC<Props> = forwardRef(
                 {
                   label: formatMessage({ id: 'page.upstream.step.healthyCheck.active' }),
                   name: ['checks', 'active'],
-                  component: <ActiveHealthCheck />,
+                  component: (
+                    <>
+                      <ActiveHealthCheck />
+                      <Divider orientation="left" plain></Divider>
```

Comment:
collapse empty tag
```suggestion
                      <Divider orientation="left" plain/>
```

Reply:
Oh yes, and we'd better add a space before / :)